### PR TITLE
[Storefront] Add profile layout

### PR DESCRIPTION
**Pull request by LEHOANGGLAM**
Add profile layout

### CODE REVIEW

File: storefront/common/components/ProfileLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head';
+import { Container } from 'react-bootstrap';
+import UserProfileLeftSideBar from './UserProfileLeftSideBar';
+import { BreadcrumbModel } from '@/modules/breadcrumb/model/BreadcrumbModel';
+import BreadcrumbComponent from './BreadcrumbComponent';
+type Props = {
+  children: React.ReactNode;
+  breakcrumb: BreadcrumbModel[];
```
Comment (khoahd7621) [inline storefront/common/components/ProfileLayout.tsx:8]: mistyped here: breakcrumb => breadcrumb

**Review by khoahd7621 — COMMENTED** [review]
Reviewed, needs a change.

**Review by nguyenvanhadncntt — APPROVED** [review]

File: storefront/common/components/ProfileLayout.tsx (hunk 1 above)
Comment (LEHOANGGLAM) [reply storefront/common/components/ProfileLayout.tsx:8]: thanks

**Review by khoahd7621 — APPROVED** [review]
Reviewed

**Pull request merged**